### PR TITLE
[jdbc] added a README

### LINKED ISSUE #459: README for jdbc binding
I have been looking into using the jdbc driver a bit and found that there is not a concise, central location for documentation on usage, properties, etc.

To really understand what was going on and how to use the jdbc binding I looked at information in a few different places:
* https://github.com/brianfrankcooper/YCSB/wiki/Using-the-Database-Libraries is a good start but does not give any information on what your database state needs to look like, or how to configure for it.
* #128 - sheds some light on where to stick your driver, not mentioned there is you can also just specify it directly in your command with -cp.
* https://github.com/brianfrankcooper/YCSB/tree/master/jdbc/src/main/resources/sql and https://github.com/brianfrankcooper/YCSB/blob/master/jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBCreateTable.java, but the usage and locations of these are not document anywhere I can find, also it does not seem that the JdbcDBCreateTable.java works for MySQL.
* properties and default values in https://github.com/brianfrankcooper/YCSB/blob/master/jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClientConstants.java

Not sure if anyone else has had issues getting it running but at the moment I think the jdbc-binding could benefit by having an intuitive README like most of the other bindings.

### PR DESCRIPTION
**Pull request by kruthar**
Fixes #459

### CODE REVIEW

File: jdbc/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+```sh
+db.driver=com.mysql.jdbc.Driver
+db.url=jdbc:mysql://127.0.0.1:3306/ycsb
+db.user=admin
+db.passwd=admin
+```
+
+Be sure to use your driver class, a valid jdbc connection string, and credentials to your database.
+
+You can add these to your workload configuration or a seperate properties file and specify it with ```-P``` or you can add the properties individually to your ycsb command with ```-p```.
```
Comment (cmccoy) [inline jdbc/README.md]: Nit picking, but spelling: "separate"

File: jdbc/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+You need to set the following connection configurations:
+
+```sh
+db.driver=com.mysql.jdbc.Driver
+db.url=jdbc:mysql://127.0.0.1:3306/ycsb
+db.user=admin
+db.passwd=admin
+```
+
+Be sure to use your driver class, a valid jdbc connection string, and credentials to your database.
```
Comment (cmccoy) [inline jdbc/README.md]: If you're modifying this anyway, could you be consistent about capitalizing JDBC?

**Comment by cmccoy** [comment]
Thanks! Two small things, if you have time.

**Comment by kruthar** [comment]
Spelling fixes committed.

**Comment by cmccoy** [comment]
Thanks!

**Comment by busbey** [comment]
this is excellent, I've wanted to close this gap for a long time. Thanks!

**Comment by busbey** [comment]
I think this takes care of the "document how to get jdbc drivers on the classpath" part of #128

**Comment by kruthar** [comment]
@busbey it should ya. I've started looking into the second part of #128 also and a few other open issues regarding JDBC, shall see if anything else comes to me.

**Pull request merged**